src/templates/template-common/utils.py

```diff
@@ -135,7 +170,19 @@ def setup_output_dir(config: Any, rank: int) -> Path:
         path.mkdir(parents=True, exist_ok=True)
         config.output_dir = path.as_posix()
 
-    return Path(idist.broadcast(config.output_dir, src=0))
+    return Path(idist.broadcast(str(config.output_dir), src=0))
```

Comment:
I do not understand why we need this change. Can you provide a valid reason ?
I'm especially interested in why previously we didn't need this modification and now we do.

Reply:
I feel like due to nature of OmegaConf it seems to convert the data type of `config.output_dir` to Posix.Path. I was getting this error in few templates in CI with Hydra so it seems better to be safe by using `__str__` representation of Posix Path